config.yaml

```diff
@@ -2,10 +2,20 @@
 # See LICENSE file for licensing details.
 
 options:
-  update-time-range:
+  restart-time-range:
     type: string
     description: >
-      Preferred UTC time range for LTS patch updates in 24 hour  format. If empty, the update will
-      take place whenever an update is available. For example, 03-05 will allow updates to take
-      place from 3AM UTC to 5AM UTC. Awaits for running job completion for 5 minutes. 
+      Preferred UTC time range for restarting Jenkins in 24 hour format. If empty, restart will
```

Comment:
This looks simpler to me:  `Preferred UTC time rangein 24 hour format  for restarting Jenkins. If empty, restart will take place if:`